### PR TITLE
Adding Hotjar Tracking ID Variable to studio and lms setting files

### PR DESCRIPTION
**Pull request by madnihamza1841**
**Description:**
Adding Hotjar Tracking IDs to cms settings. 

**JIRA:**
No associated ticket

**Testing instructions:**
1. Check that the Hotjar is showing user data for green dev

### CODE REVIEW

File: cms/envs/production.py [hunk 1]
```diff
@@ -551,3 +551,9 @@ def get_env_setting(setting):
 ########################## Derive Any Derived Settings  #######################
 
 derive_settings(__name__)
+
+# HOTJAR ANALYTICS
+HOTJAR_TRACKING_ID = ENV_TOKENS.get(
+    'HOTJAR_TRACKING_ID',
+    HOTJAR_TRACKING_ID
```
Comment (taimoor-ahmed-1) [inline cms/envs/production.py:558]: Please replace default value with `None`

File: lms/envs/production.py [hunk 2]
```diff
@@ -679,7 +679,10 @@ def get_env_setting(setting):
 GOOGLE_SITE_VERIFICATION_ID = ENV_TOKENS.get('GOOGLE_SITE_VERIFICATION_ID')
 
 # HOTJAR ANALYTICS
-HOTJAR_TRACKING_ID = ENV_TOKENS.get('HOTJAR_TRACKING_ID')
+HOTJAR_TRACKING_ID = ENV_TOKENS.get(
+    'HOTJAR_TRACKING_ID',
+    HOTJAR_TRACKING_ID
```
Comment (taimoor-ahmed-1) [inline lms/envs/production.py:684]: Please replace `HOTJAR_TRACKING_ID` with None

File: cms/envs/production.py (hunk 1 above)
Comment (madnihamza1841) [reply cms/envs/production.py]: Done

File: lms/envs/production.py (hunk 2 above)
Comment (madnihamza1841) [reply lms/envs/production.py]: Done

**Review by taimoor-ahmed-1 — APPROVED** [review]

**Review by burhandodhy — APPROVED** [review]

**Pull request merged**